libpromises/parser_state.h

```diff
@@ -59,6 +59,7 @@ typedef struct
     bool references_body;
 
     char *promiser;
+    Rval promiser_attribute;
```

Comment:
Comments missing. I don't like the "promiser_attribute" name, "promiser" is opposite of "attribute" in my head. 

A better name would be "promiser_as_rval", but then it must not be a new pseudo-constraint in the syntax, just an internal field of our Promiser struct.